Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -26,7 +26,7 @@ def bump_args
       switch "--cask", "--casks",
              description: "Check only casks."
       switch "--open-pr",
-             description: "Open a pull request for the new version if there are none already open."
+             description: "Open a pull request for the new version if none have been opened yet."
```

Comment:
This is my attempt at clarifying that matching open or closed PRs will block you from opening another one. It's still probably too subtle though.